From 21b4983c070ef0c6418e28a2e70f17fe4878a646 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Mon, 4 May 2026 16:37:45 +0800
Subject: [PATCH] configurations: nvidia: cx9_mezzanine_module: add data drive
 temp sensor

Add NVMeMI sensor node to support data drive temperature monitoring.

[1] https://gerrit.openbmc.org/c/openbmc/entity-manager/+/89949/2

Change-Id: Ic6cc346daea73b6cf9c9582b40a0d547bfe1b720
Signed-off-by: Potin Lai <potin.lai@quantatw.com>
---
 .../nvidia/cx9_mezzanine_module.json          | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/configurations/nvidia/cx9_mezzanine_module.json b/configurations/nvidia/cx9_mezzanine_module.json
index 0a8b6f1..efba418 100644
--- a/configurations/nvidia/cx9_mezzanine_module.json
+++ b/configurations/nvidia/cx9_mezzanine_module.json
@@ -91,6 +91,28 @@
             "Timeout": 30.0,
             "Type": "ExternalSensor",
             "Units": "Watts"
+        },
+        {
+            "Address": "$bus - 73 * 10 + 35",
+            "Name": "IOB$bus - 73 DATADRIVE_TEMP_C",
+            "PowerState": "On",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Hysteresis": 1.4,
+                    "Name": "upper critical",
+                    "Severity": 1,
+                    "Value": 70
+                },
+                {
+                    "Direction": "less than",
+                    "Hysteresis": 0.1,
+                    "Name": "lower critical",
+                    "Severity": 1,
+                    "Value": 5
+                }
+            ],
+            "Type": "NVMeMI"
         }
     ],
     "Name": "NVIDIA VR NVL72 IO Board $bus - 73",
-- 
2.53.0-Meta

